package.json

```diff
@@ -54,6 +54,7 @@
     "stream": "npm:stream-browserify",
     "typeface-roboto": "*",
     "uuid": "^9.0.1",
+    "webpack": "^5.89.0",
```

Comment:
Was this intended?  It shouldn't be necessary, but if you ran into a problem then we can create a separate issue for it.